packages/gafl-webapp-service/README.md

```diff
@@ -8,34 +8,37 @@ To run from this directory:
+| SALES_API_TIMEOUT_MS              | The timeout in milliseconds requests to the api                  |    no    | 10000                                                     |                               |
+| GOV_PAY_API_URL                   | The GOV.UK Pay API base url                                      |    no    | Yes                                                       |                               |
+| GOV_PAY_APIKEY                    | GOV pay access identifier                                        |    no    | Yes                                                       |                               |
+| GOV_PAY_REQUEST_TIMEOUT_MS        | Timeout in milliseconds for API requests                         |    no    | Yes                                                       |                               |
+| FEEDBACK_URI                      | Location of feedback survey                                      |    no    | #                                                         |                               |
+| ANALYTICS_PRIMARY_PROPERTY        | Analytics ID for tracking inc ecommerce                          |    no    |                                                           |                               |
+| ANALYTICS_XGOV_PROPERTY           | Analytics ID for page view tracking                              |    no    |                                                           |                               |
+| ATTRIBUTION_REDIRECT              | Target for redirect of attribution endpoint                      |    no    | /                                                         |                               |
+| SERVICE_PAGE                      | GOV.UK service page                                              |    no    | https://www.gov.uk/fishing-licences/buy-a-fishing-licence |                               |
+| AIRBRAKE_HOST                     | URL of airbrake host                                             |    no    |                                                           |                               |
+| AIRBRAKE_PROJECT_KEY              | Project key for airbrake logging                                 |    no    |                                                           |                               |
+| SHOW_WELSH_CONTENT                | Display option to change to welsh language                       |    no    |                                                           |                               |
+| ENABLE_ANALYTICS_OPT_IN_DEBUGGING | Set log if analytics been checked in non-production              |    no    |                                                           |                               |
+| ERROR_PAGE_ROUTE                  | Display error pages to support welsh language                    |    no    |                                                           |                               |
```

Comment:
Only meant you to add an entry for `ERROR_PAGE_ROUTE` rather than the other two, as they're going to be deleted soon, but we'll leave it as-is for now. Thanks.